### ISSUE TITLE
transferable streams is missing

### ISSUE THREAD

**Issue by jimmywarting** [microsoft/TypeScript#38723]
The error i got from using postMessage was:

```
  Overload 1 of 2, '(message: any, transfer: Transferable[]): void', gave the following error.
    Type 'ReadableStream<any>' is not assignable to type 'Transferable'.
      Type 'ReadableStream<any>' is missing the following properties from type 'OffscreenCanvas': height, width, convertToBlob, getContext, and 4 more.
```

https://chromestatus.com/feature/5298733486964736

**Comment by DanielRosenwasser** [comment]
Can you provide a minimal repro?

**Comment by jimmywarting** [comment]
```js
var stream = new ReadableStream()
var stream2 = new ReadableStream()
new MessageChannel().port1.postMessage(stream, [stream])
window.postMessage(stream2, '*', [stream2])
```

**Comment by IllusionMH** [comment]
Is it a really bug at this moment?
Because at this point it looks like only Chrome has this feature and it is behind a flag.

**Comment by jakebailey** [comment]
This was fixed in #50884.
